runner/executor.go

```diff
@@ -64,7 +64,7 @@ func (runner CommandRunner) Run(ctx task.InputContext) *task.Result {
 	return result
 }
 
-func Execute(pluginVersion string, buildDate string, runner Runner) {
+func execute(pluginVersion string, buildDate string, runner Runner) {
```

Comment:
Won't this create problems if server input changes on the task? Since we only deserialize it once when creating command factory. Or if there are tasks that run parallel and use different argo servers for example.

Reply:
I was tempted not to answer since you are on vacation :P 

Anyhow, this change should not be the problem because in runner we have factoryBuilder which is actually a reference to type of FactoryBuilder which is then a function which is called with a new context each time we execute.

In the end consecutive executions have different context which results in different servers/clients.